socket.c

```diff
@@ -240,6 +271,24 @@ void socket_listen(struct thread *t)
         struct addrinfo *ai = getaddrinfo_or_die(opts->host, opts->port, &hints,
                                                  cb);
         int port = atoi(opts->port);
+#if defined(WITH_TCPDEVMEM_CUDA) || defined(WITH_TCPDEVMEM_UDMA)
+        /* TCP Devmem:
+         * Since each thread has a CUDA buffer, and
```

Comment:
Since this is guarded by CUDA || UDMA, then it should be 'dmabuf', not 'cuda buffer', I think.